cumulusci.yml

```diff
@@ -655,6 +680,35 @@ tasks:
                 u.LanguageLocaleKey = languageMap.get('%%%PARAM_1%%%');
                 Update u;
 
+    assign_permset:
```

Comment:
I think we can switch to this:

https://cumulusci.readthedocs.io/en/latest/tasks.html#create-permission-set


Especially given this update which will launch next week:

https://github.com/SFDO-Tooling/CumulusCI/pull/2115

Reply:
Very nice, especially `create_bulk_data_permission_set`.